### PR TITLE
Autopsies No Longer Need Surgery

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Autopsies no longer require surgery to do.

It would be way too snowflakey to give specific exceptions to surgery, but we can go with the next best thing: removal of the surgery aspects of autopsies.

You will still have to put someone on an operating table (or regular table) in order to perform the autopsy, but you will no longer need to cut open every single body part in order to scan it.

If you're looking for some 'mai immershun' explanation, then it's the year 2556, scanners are good or something now.

Also, screw autopsies, autopsy code, autopsy data, trace chems, and the wound system--it's the shitocode to end all shitcode.

:cl: Fox McCloud
tweak: Autopsies can now be performed without needing to cut someone open
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/autopsy.dm [hunk 1]
```diff
@@ -159,24 +153,9 @@
 	P.info = "<tt>[scan_data]</tt>"
 	P.overlays += "paper_words"
 
-	if(istype(usr,/mob/living/carbon))
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/autopsy.dm]: Old code shitcode, right here.

File: code/game/objects/items/devices/autopsy.dm [hunk 2]
```diff
@@ -159,24 +153,9 @@
...
-			P.layer = 20
-			P.plane = HUD_PLANE
-		else if(!usr.l_hand)
-			P.loc = usr
-			usr.l_hand = P
-			P.layer = 20
-			P.plane = HUD_PLANE
-
-	if(istype(usr,/mob/living/carbon/human))
-		usr:update_inv_l_hand()
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/autopsy.dm]: Really, you couldn't even be bothered to typecast. Granted, this has 100% safe, but *really*....***really***.

**Comment by Aurorablade** [comment]
*gets the sedative for fox*

**Comment by Jovaniph** [comment]
Should also just remove Coroner as an alternative job title as well if it now that easy.

**Comment by Fruerlund** [comment]
This is good, we should not remove coroner. We should keep custom job titles to allow people to roleplay their characters and professions

**Comment by Fox-McCloud** [comment]
>Should also just remove Coroner as an alternative job title as well if it now that easy.

It was already easy; you just don't have to click+confirm+wait forever anymore to get from A to B. 

Anyone who could have performed an autopsy before can perform one now, and anyone who couldn't before still couldn't.

This doesn't expand availability to anyone; it just makes autopsies less time consuming to perform.

**Comment by SamHPurp** [comment]
@Fruerlund imo, seperating it out as its own role and expanding on the mechanics would be great.

Add a requests console to the room too so you can mail the HoP dead guys IDs/PDAs... I smell a PR coming on...

**Comment by Dinarzad** [comment]
Bless this PR.
Bless wit with the Honkmother, Nar'sie and whatever other fucking deity there is.

**Comment by taukausanake** [comment]
offtopic because it needs to be said now. Honkmother cult

**Comment by Lady-Luck** [comment]
Oh hey, this is a really good idea.

**Comment by PhantasmicDream** [comment]
Yay, don't have to worry about people stealing the scalpel from the morgue anymore.

**Comment by Anasari** [comment]
FInally, autopsy is no longer tedious and bothersome. +1 <3

File: code/game/objects/items/devices/autopsy.dm (hunk 1 above)
Comment (Crazylemon64) [inline code/game/objects/items/devices/autopsy.dm]: wats a helper proc

File: code/game/objects/items/devices/autopsy.dm [hunk 3]
```diff
@@ -185,23 +164,18 @@
 
 	if(target_name != M.name)
```
Comment (Crazylemon64) [inline code/game/objects/items/devices/autopsy.dm]: This should probably be `realname` so doofy stuff doesn't happen if you put gas masks on the cadavers but eh that's a matter for another day

**Pull request merged**